.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data requested**
+A clear and concise description of the data needed
+
+**How would this data be visualized?**
+
+
+**Which countries is this data related to?**
+A clear and concise description of the countries this data is related to
+
+**Which topics is this data related to?**
+Which [topics](https://takwimu.africa/) does this data fall under
```

Comment:
From "Also in "help text", should provide the options for these two." I meant list out the options to select from in the help text in these two options above (country and topic). i.e list out the countries to select from, and the topics too (probably with an "other" for topics).